README.md

```diff
@@ -1,98 +1,129 @@
+
+### Enable developer mode on Windows
+
 https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development
 
-### Address Unsigned Powershell Scripts
+### Addressing Unsigned Powershell Scripts
 FactoryOrchestrator contains a series of unsigned powershell scripts. Windows security measures prevent unsigned scripts from executing. In order to develop on FactoryOrchestrator, you need to do one of two things.
 
-#### Self-sign
-More information on how to do this can be found here: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_signing?view=powershell-7
+1. Self-sign the scripts
+
+   More information on how to do this can be found here: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_signing?view=powershell-7
+
+2. Set the Execution Policy to Unrestricted
+   
+    This method is not recommended. Setting the Execution Policy to Unresricted allows any powershell script to run. 
 
-#### Set the Execution Policy to Unrestricted
-This method is not recommended. Setting the Execution Policy to Unresricted allows any powershell script to run. 
-Documentation on Execution Policy:
+    Documentation on Execution Policy:
 https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7
 
-## 3. Building
 When building the source code, keep in mind that certain actions only occur when the build is run as part of an Azure DevOps Pipeline or when run in the "Release" configuration locally. For example, [DocFX documentation](https://dotnet.github.io/docfx/) for the Core and Client Libraries is only generated when the build is run in in one of those two modes. These actions are skipped in Debug builds to increase inner-dev-loop speed.
 
-## 4. Debugging
-Run _FactoryOrchestratorApp (Universal Windows)_ and _FactoryOrchestratorService_ in separate Visual Studio 2019+ instances. This will allow the app and the service to communicate with each other. 
+## Debugging
+
+Run _FactoryOrchestratorApp (Universal Windows)_ and _FactoryOrchestratorService_ in separate Visual Studio 2019+ instances. This will allow the app and the service to communicate with each other.
 Even if you are coding for just the app, you generally need to run the service so the app has something to connect to. The service must be run as administrator.
 
 At the moment, running either the app or service under a debugger will cause exceptions to be thrown related to certain registry keys (ex: GetOEMVersionString()), but all exceptions are caught and will not result in a crash. This is because they try to access registry keys specific to a Microsoft internal product. You can routinely skip over these exceptions when they occur or disable them.
```

Comment:
This is no longer true, can you remove this for me :)?

Reply:
hmmm i can't really tell what 'this' refers to:
![image](https://user-images.githubusercontent.com/1374317/95811108-6526d100-0cc7-11eb-80b9-6ebd95235777.png)

might be me tho...

Reply:
oh maybe the whole thing.. 😮